### ISSUE TITLE
Error: Cannot find module 'node:child_process'

### ISSUE THREAD

**Issue by IWANABETHATGUY** [rome/tools#2606]
### Environment information

```bash
Node.js v14.17.3
```

### What happened?

Node throw error
```bash
Error: Cannot find module 'node:child_process'
```
 when run `rome format`

### Expected result

No error

### Code of Conduct

- [X] I agree to follow Rome's Code of Conduct

**Comment by IWANABETHATGUY** [comment]
This is feature are largely used and stable until https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V18.md#2022-05-03-version-1810-current-targos, so we need to better remove 'node:' prefix for now.

**Comment by jonahsnider** [comment]
Upgrading to Node.js v14.18.0 or higher will [fix your issue](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V14.md#2021-09-28-version-14180-fermium-lts-targos).

**Comment by IWANABETHATGUY** [comment]
> 
Yeah, I know that, but It would be better to remove `node:` prefix. Because  it is not easy to upgrade nodejs version in some CI environment due to some legacy issue.

**Comment by ematipico** [comment]
These are scripts that are used inside **our** CI environment, which supports the usage of the `node:` prefix. If you tried to use a different CI, you can do it by removing the prefix locally; I don't think we need to remove the prefix from our source code.

To be frank, I am thinking to bump all our scripts to use node 16. If that happens, it means that's a bug? What's your use case that you can't run the script?

**Comment by IWANABETHATGUY** [comment]
> These are scripts that are used inside **our** CI environment, which supports the usage of the `node:` prefix. If you tried to use a different CI, you can do it by removing the prefix locally; I don't think we need to remove the prefix from our source code.
> 
> To be frank, I am thinking to bump all our scripts to use node 16. If that happens, it means that's a bug? What's your use case that you can't run the script?

You mean remove these `node:` prefix everytime that code inside `node_modules` ? You maybe lose potential user that  nodejs version under `14.18.0`.

**Comment by IWANABETHATGUY** [comment]
Also IIRC, there should no difference between add `node:` prefix or not with a Nodejs builtint module.

**Comment by ematipico** [comment]
> > These are scripts that are used inside **our** CI environment, which supports the usage of the `node:` prefix. If you tried to use a different CI, you can do it by removing the prefix locally; I don't think we need to remove the prefix from our source code.
> > To be frank, I am thinking to bump all our scripts to use node 16. If that happens, it means that's a bug? What's your use case that you can't run the script?
> 
> You mean remove these `node:` prefix everytime that code inside `node_modules` ? You maybe lose potential user that nodejs version under `14.18.0`.

OK, I now see the problem.

Then this means that the only place where we want to remove the prefix is inside `postinstall.js`, where use `node:fs`. The rest of the scripts are executed in our CI environment and won't affect users. Does it make sense?

**Comment by IWANABETHATGUY** [comment]
> > > These are scripts that are used inside **our** CI environment, which supports the usage of the `node:` prefix. If you tried to use a different CI, you can do it by removing the prefix locally; I don't think we need to remove the prefix from our source code.
> > > To be frank, I am thinking to bump all our scripts to use node 16. If that happens, it means that's a bug? What's your use case that you can't run the script?
> > 
> > 
> > You mean remove these `node:` prefix everytime that code inside `node_modules` ? You maybe lose potential user that nodejs version under `14.18.0`.
> 
> OK, I see the problem now.
> 
> Then this means that the only place where we want to remove the prefix is inside `postinstall.js` and `rome`, where use `node:fs` and `node:child_process`. The rest of the scripts are executed in our CI environment and won't affect users. Does it make sense?

Yeah, you got it.
